templates/CSI/rbd/pod.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: v1
+kind: Pod
```

Comment:
Probably it can stay in templates/app-pod/pod.yaml  and both fs and rbd can utilize it, I see its duplicated for fs and rbd.

Reply:
@vasukulkarni yeah this is a good suggestion. Shall I create a directory named app-pod under templates ?

Reply:
@nehaberry yes please go ahead, thanks!